src/control/lib/spdk/SConscript

```diff
@@ -28,7 +28,9 @@ def scons():
 
     senv.nc = senv.Object("src/nvme_control.c")
     senv.ncc = senv.Object("src/nvme_control_common.c")
-    denv.nvmecontrol = senv.StaticLibrary("nvme_control", [senv.nc, senv.ncc],
+    senv.cc = senv.Object("../../../common/control.c")
```

Comment:
I have a minor preference for exporting the file

something like control_tgts = File("control.c")
Export ('control_tgts')

then 
Import('control_tgts') # note that tgts suffix will avoid any issues with pylint checking.
and pass that to your compile line

Reply:
done